provider/oracle/environ_test.go

```diff
@@ -201,9 +201,36 @@ func (e *environSuite) TestAdoptResources(c *gc.C) {
 	c.Assert(err, gc.IsNil)
 }
 
+func (e *environSuite) assertSetupStopInstances(c *gc.C) (*oracle.OracleEnviron, string) {
+	testEnvironAPI := oracletesting.DefaultEnvironAPI
+	env, err := oracle.NewOracleEnviron(
+		&oracle.EnvironProvider{},
+		environs.OpenParams{
+			Config: testing.ModelConfig(c),
+		},
+		testEnvironAPI,
+		&advancingClock,
+	)
+	c.Assert(err, gc.IsNil)
+
+	hostname, err := oracle.CreateHostname(env, "0")
+	c.Assert(err, gc.IsNil)
+	testEnvironAPI.FakeInstance.All.Result[0].Name = fmt.Sprintf("/Compute-a432100/sgiulitti@cloudbase.com/%s/ebc4ce91-56bb-4120-ba78-13762597f837", hostname)
+	oracle.SetEnvironAPI(env, testEnvironAPI)
+	return env, hostname
+}
+
+func repairEnv(env *oracle.OracleEnviron) {
+	testEnvironAPI := oracletesting.DefaultEnvironAPI
+	testEnvironAPI.FakeInstance.All.Result[0].Name = "/Compute-a432100/sgiulitti@cloudbase.com/0/ebc4ce91-56bb-4120-ba78-13762597f837"
```

Comment:
Do we have to reset this to "/blah/0/uuid"?
Can we just set Name to a CreateHostname() when the FakeInstance is first set up?
It seems a little messy.

Reply:
Move the name change to SetUpTest(), removing necessity of the two helper methods.